core/utilities/LogManager.cpp

```diff
@@ -227,15 +228,20 @@ std::vector<std::string> LogManager::getLogsToUpload()
 
     std::vector<std::string> remoteFiles = m_logUploader->getRemoteLogs();
 
-    logFiles.erase(std::remove_if(logFiles.begin(), logFiles.end(),
-                                  [&](const std::string& file) -> bool {
-                                      return std::find(remoteFiles.begin(), remoteFiles.end(), file) !=
-                                             remoteFiles.end();
-                                  }),
-                   logFiles.end());
+    for (auto log : remoteFiles)
+    {
+        log = wolkabout::StringUtils::removePrefix(log, m_logDirectory + "/");
+    }
```

Comment:
This for loop is the only addition in this file, the rest is just CLang formatting